server/main.js

```diff
@@ -5,7 +5,14 @@ import bodyParser from 'body-parser';
 /************************************************
 * Setup Winston Logger
 ************************************************/
-const logger = require('winston');
+const { createLogger, transports, format } = require('winston');
```

Comment:
I think Richard said he wasn't sure if these changes were necessary. Have we tried running without, and/or confirming our logging still works how we expect?

Reply:
**Would like to address this before merging!**